Update README (#4)
Gamify-IT/lecturer-interface: README.md

# Lecturer-Interface

> A easy to use interface with that the lecturers can configure the areas of the overworld and the configurations of the minigames.

Reviewer: ```suggestion
An easy to use interface that lecturers can use to configure the areas of the overworld and the shown minigames.
```
